lib/markdowngen.ml

```diff
@@ -90,7 +90,7 @@ let of_args args =
     match is_in, arg.Param.typedef.ty with
     | false, Unit -> []
     | _ ->
-      let name = arg.Param.name in
+      let name = match arg.Param.name with Some s -> s | None -> "unnamed" in
```

Comment:
I see what you meant when we were discussing this. It may be worth to abstract some of these commonly used functions in a "utils" module